ngraph/python/src/ngraph/opset6/ops.py

```diff
@@ -58,6 +58,35 @@
 # -------------------------------------------- ops ------------------------------------------------
 
 
+@nameable_op
+def ctc_greedy_decoder_seq_len(
+        data: NodeInput,
+        sequence_length: NodeInput,
+        blank_index: Optional[NodeInput] = None,
+        merge_repeated: bool = True,
+        classes_index_type: str = "i32",
+        sequence_length_type: str = "i32",
+        name: Optional[str] = None,
+) -> Node:
+    """Return a node which performs CTCGreedyDecoderSeqLen.
+
+    @param data:            The input 3D tensor. Shape: [batch_size, seq_length, num_classes]
+    @param sequence_length: Input 1D tensor with sequence length. Shape: [batch_size]
+    @param blank_index:     Scalar or 1D tensor with specifies the class index to use for the blank class.
+                            Optional parameter. Default value is num_classes-1.
+    @return:                The new node which performs CTCGreedyDecoderSeqLen.
+    """
+    inputs = as_nodes(data, sequence_length, blank_index)
+
+    attributes = {
+        "merge_repeated": merge_repeated,
+        "classes_index_type": classes_index_type,
+        "sequence_length_type": sequence_length_type
+    }
```

Comment:
I assume all tests for `blank_index = False` are failing due to recognizing `None` value as `dtype('O')`. You should add some check before setting inputs, please add `if blank_index is None` and assign default value of `num_classes-1` to it if true.